.github/workflows/post-merge.yml

```diff
@@ -323,6 +336,40 @@ jobs:
           echo "Building the code"
           make build
 
+      - name: Upload Assets
+        if: ${{ inputs.run_upload_assets}}
+        working-directory: ${{ inputs.project_folder }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.SYS_ORCH_GITHUB }}
+          ASSETS_DIR: "${{ inputs.assets_dir }}"
+        run: |
+          # Check if ASSETS_DIR is provided and exists
+          if [ -z "$ASSETS_DIR" ] || [ ! -d "ASSETS_DIR" ]; then
+            echo "Error: ASSETS_DIR is not set or does not exist."
+            exit 1
+          fi
+
+          cd ${{ inputs.project_folder }}
+
+          # Fetch the latest tag using the project folder
+          latest_tag=$(git tag --list "${{ inputs.project_folder }}/*" | sort -V | tail -n 1)
```

Comment:
`${{ inputs.project_folder }}` should be an environment variable.

The reason being, user inputs are not santized, and can harbor commands which get evaluated at runtime.  For example, if `inputs.project_folder` is set to `$(rm -rf /*)`, bad things happen.